.circleci/config.yml

```diff
@@ -237,6 +240,12 @@ jobs:
       <<: *incubator_helm_channel
     <<: *helm_push
 
+  helm_push_nightly:
+    <<: *default_doks_image
+    environment:
+      <<: *nightly_helm_channel
+    <<: *helm_push
```

Comment:
Won't you hit the same problem here? The second night the version given by git-version is the same because de commit is the same.

`helm push --version "$(cat .version)-NIGHTLY"`

Reply:
fixed in the next commit.